packages/node/src/indexer/store.service.ts

```diff
@@ -14,19 +23,85 @@ export class StoreService {
 
   async syncSchema(
```

Comment:
store service don't need to know about graphql schema. I suggest you can make syncSchema() accept models and relations.